e2e/atlas/access_lists_test.go

```diff
@@ -137,4 +137,24 @@ func TestAccessList(t *testing.T) {
 		}
 		a.True(found)
 	})
+
+	t.Run("Create Delete After with CurrentIp", func(t *testing.T) {
+		cmd := exec.Command(cliPath,
+			atlasEntity,
+			accessListEntity,
+			"create",
+			"--currentIp",
+			"--deleteAfter="+time.Now().Add(time.Minute*time.Duration(5)).Format(time.RFC3339),
+			"--comment=test",
+			"-o=json")
+		cmd.Env = os.Environ()
+		resp, err := cmd.CombinedOutput()
+		req.NoError(err)
+
+		var entries *mongodbatlas.ProjectIPAccessLists
+		err = json.Unmarshal(resp, &entries)
+		req.NoError(err)
+
+		a.NotEmpty(entries.Results)
```

Comment:
deleting after 5 minutes is not a wise decision in my opinion as e2e tests may run on the same machine which was previously used to run this command. In addition, having an access list entry may prevent deleting the project if needed. I would split this into two-step and call the delete command to delete the entry immediately.

Reply:
Sure! I thought it should be fine given that our last test before was exactly testing this feature.